python/xoscar/collective/gloo/xoscar_pygloo.pyi

```diff
@@ -0,0 +1,199 @@
+import datetime
+from ctypes import c_void_p
+from enum import IntEnum
+from typing import Callable, List, Optional
+
+import xoscar_pygloo
+
+class ReduceOp(IntEnum):
+    SUM = 0
+    PRODUCT = 1
+    MIN = 2
+    MAX = 3
+    BAND = 4
+    BOR = 5
+    BXOR = 6
+    UNUSED = 7
+
+class GlooDataType(IntEnum):
```

Comment:
Should this be ``GlooDataType_t``  ? Here should be consistant with the class name you bind. Please check all the other places.

Reply:
ok, I have changed it